### ISSUE TITLE
Incompatible with 7.22.2

### ISSUE THREAD

**Issue by Narev** [mendixlabs/list-view-controls#40]
When checking my console it tells me the list view controls are not compatible with with Mendix version 7.22.2  I also noticed that the initial filtering does not work. e.g. when you set a dropdown filter to be the default it does not apply correctly on load. After selecting the filter explicitly it does work as expected.

**Comment by JeffreyMansystems** [comment]
This applies as well for version 7.23.2. When applying the pagination widget for the list view it re-renders the data, and causing a move to top of page.

**Comment by Andries-Smit** [comment]
Hi Pieter and Jeffrey. Thank you for reporting this issue. We will add some time our next sprint to look at this issue. Cheers, Andries

**Comment by Andries-Smit** [comment]
Hi Pieter & Jeffry

@Narev I upgrade the project to 7.22.2 though i dont see any console error message
Nor an issue with the initial filter: see https://listviewcontrols.mxapps.io/p/defaultfilter
Could you please share a sample project where i can reproduce the issue and solve the issue? 

@JeffreyMansystems 
I dont understand when exactly you get this issue. please provide us a sample project

Thanks

**Comment by JeffreyMansystems** [comment]
@Andries-Smit, I checked the widget and page where I've used the widget. In the previous version of my project I was using a template grid and replaced with a list view. In that template grid I have containers with conditional visibility on it. The visibility can be toggled from within the list view. When using a template grid I had to refresh the single object, but I figured out that that is not necessary any more when using a list view.

So for me the only thing is, that when I display the page with a list view control widget on it in a Chrome browser (other browsers not tested) it gives me an error in the console: 
"This Mendix version is not compatible with list view controls. The List view controls use is limited."
I'm now using MX 7.23.3
